### DISCUSSION TITLE
Subscription refund

### DISCUSSION THREAD

**Discussion by GustavoAgCosta** [aclap-dev/video-downloadhelper#1001] [License & Purchase requests]
### Your question

I would like to request a refund for my subscription made yesterday, 7/11, as there were several videos that I was unable to download through the app.
![Screenshot_1](https://github.com/user-attachments/assets/51bebdf7-3729-4663-9403-d0fe93037f12)
email: gustavocosta2001@gmail.com

### Addon & CoApp version and details

I would like to request a refund for my subscription made yesterday, 7/11, as there were several videos that I was unable to download through the app.
![Screenshot_1](https://github.com/user-attachments/assets/51bebdf7-3729-4663-9403-d0fe93037f12)
email: gustavocosta2001@gmail.com
![Screenshot_1](https://github.com/user-attachments/assets/7ffe1fd9-5ec8-472d-b169-a9c783486942)

**Comment by Storm0099** [comment]
I was going to pay today but after that I'm really hesitant especially since there's no response from support

**Reply by Mattmjs** [reply]
A licensed doesn't guarantee that the download will succeed as opposed to using it for free. What it does is removing the 2 hour limit and removing the need for a watermark on
YouTube. That's the main reasons for getting a
license. 

Try it for free first on the sites you need it for,
if it works then you can consider getting the licence.

**Comment by Storm0099** [comment]
Thanks I tried to download now but it's not working unfortunately

في الخميس، ٨ أغسطس ٢٠٢٤, ١١:٠٥ Mattmjs ***@***.***> كتب:

**Reply by Mattmjs** [reply]
You should open a new discussion if you want it investigated with your version information and a link where it fails.

**Comment by Storm0099** [comment]
Hello, I have a problem with the tool... When I put the activation key in
it, it does not activate, but rather gives me loading and does not finish

بتاريخ الجمعة، 9 أغسطس 2024، جاء من Mattmjs ***@***.***>:

**Reply by jcfrog** [reply]
Hi,
What do you see on your versions about the CoApp? https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by Storm0099** [comment]
I don't want a refund, I just want to present the problem to you and I
want a solution.   This is what appears when I enter the activation
key...it takes an infinite amount of time to download

بتاريخ الخميس، 27 مارس 2025، جاء من jcfrog ***@***.***>:

**Reply by jcfrog** [reply]
Something is blocking downloadhelper process, I recommend to have a look at your antivirus or firewall settings.
